packages/one-app-runner/src/startApp.js

```diff
@@ -77,7 +77,14 @@ module.exports = async function startApp({
     return command;
   };
 
-  const command = `docker pull ${appDockerImage} && docker run -t -p 3000-3005:3000-3005 -e NODE_ENV=development ${generateEnvironmentVariableArgs(envVars)} ${generateModuleMountsArgs(modulesToServe)} ${appDockerImage} /bin/sh -c "${generateServeModuleCommands(modulesToServe)} ${generateSetMiddlewareCommand(parrotMiddlewareFile)} ${generateSetDevEndpointsCommand(devEndpointsFile)} node lib/server/index.js --root-module-name=${rootModuleName} --module-map-url=${moduleMapUrl} ${generateUseMocksFlag(parrotMiddlewareFile)}"`;
+  const generateModuleMap = () => {
+    if (moduleMapUrl) {
```

Comment:
ternary operator anyone? 😂JKD I'm not fussed about that...

Reply:
@JamesSingleton 
```suggestion
    return moduleMapUrl ? `--module-map-url=${moduleMapUrl}` : '';
```